store/cart.ts

```diff
@@ -1,118 +1,34 @@
-            pickPrice: {
-              currencyIso: 'SEK',
-              formattedValue: '11:50',
-              priceType: 'BUY',
-              value: 11.5
-            },
-            unit: {
-              name: 'st'
-            },
-            url:
-              '/handla/varor/mejeri-agg/mjolk-fil/mjolk/mellanmjolk-eko-7300170062642',
-            variances: []
-          },
-          quantity: 1,
-          replace: true,
-          totalPrice: {
-            currencyIso: 'SEK',
-            formattedValue: '11:50',
-            priceType: 'BUY',
-            value: 11.5
-          }
+    let axiosConfig = {}
+    if (config.user !== 'anonymous') {
+      axiosConfig = {
+        headers: {
+          Authorization: `Bearer ${config.authToken}`
         }
-      ],
-      extraAmountToBeReserved: {
-        currencyIso: 'SEK',
-        formattedValue: '4:00 kr',
-        priceType: 'TOTAL',
-        value: 4.0
-      },
-      guid: 'a6360054-821c-402d-bf85-3e87a4d6d8e5',
-      isPartOfSubscription: false,
-      postCode: '17261',
-      timeWindow: {},
-      totalDepositSum: {
-        currencyIso: 'SEK',
-        formattedValue: '0:00 kr',
-        priceType: 'TOTAL',
-        value: 0.0
-      },
-      totalItems: 1,
-      totalPrice: {
-        currencyIso: 'SEK',
-        formattedValue: '11:50 kr',
-        priceType: 'TOTAL',
-        value: 11.5
-      },
-      totalPriceWithExtraAmountToReserve: {
-        currencyIso: 'SEK',
-        formattedValue: '15:50 kr',
-        priceType: 'TOTAL',
-        value: 15.5
-      },
-      totalPriceWithTax: {
-        currencyIso: 'SEK',
-        formattedValue: '11:50 kr',
-        priceType: 'TOTAL',
-        value: 11.5
-      },
-      totalQuantity: 1,
-      totalTaxValues: [
-        {
-          appliedValue: {
-            currencyIso: 'SEK',
-            formattedValue: '1:23 kr',
-            priceType: 'TOTAL',
-            value: 1.23
-          },
-          value: 12.0
-        }
-      ],
-      amountMissing: {
-        currencyIso: 'SEK',
-        formattedValue: '488:50 kr',
-        priceType: 'TOTAL',
-        value: 488.5
-      },
-      coopStore: {
-        code: '016001',
-        enova: false,
-        foodBagsOnly: false,
-        name: 'STOCKHOLM'
-      },
-      isExtraOrderCart: false,
-      recipesGroups: [],
-      replaceAll: true
-    }
-  },
-  mutations: {
-    setCart(state, payload) {
-      state = payload
+      }
     }
+
+    await this['$axios']
+      .get(
+        `https://www.coop.se/ws/v2/coop/users/${config.user}/carts/${config.cartguid}?fields=DEFAULT`,
+        axiosConfig
+      )
+      .then((response) => {
+        commit('setCart', response.data)
+      })
+      .catch((error) => {
+        // eslint-disable-next-line no-console
+        console.error(error)
+      })
   }
 }
 
-export default cardsModule
+export const mutations = {
+  setCart(state, payload) {
+    // `state = payload` doesn't work
```

Comment:
```suggestion
```

Reply:
I was thinking this was a bug or that it should be able to be solved(?)

Reply:
It's one of the limitations with Vue reactivity so I don't think we will change it back